core/server/middleware/theme-handler.js

```diff
@@ -60,6 +60,8 @@ themeHandler = {
 
         // clear the view cache
         blogApp.cache = {};
+        // reset the asset hash
+        config.assetHash = null;
```

Comment:
correct me i am wrong!

aren't both lines redundant?
https://github.com/TryGhost/Ghost/blob/master/core/server/api/index.js#L80

There are two cases:
1. you activate a theme
2. you upload and activate a theme

But both are using the settings endpoint to set the new active theme.

So https://github.com/TryGhost/Ghost/blob/master/core/server/middleware/theme-handler.js#L99 this is false when loading the blog. It will set the active theme in our express app https://github.com/TryGhost/Ghost/blob/master/core/server/middleware/theme-handler.js#L85 and with your new change the `assetHash` get's refreshed.